apps/token/lib/token.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Token do
```

Comment:
I would put this module and the other ones under some namespace so you avoid any conflicts with any other code out there.